src/js/themes/minimal.js

```diff
@@ -0,0 +1,230 @@
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaThemeElement } from '../media-theme-element.js';
+
+const html = (raw, ...keys) => String.raw({ raw }, ...keys);
+
+const template = document.createElement('template');
+template.innerHTML = html`
+  <style>
+    media-controller {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+    }
+
+    media-control-bar {
+      height: 42px;
+      border-radius: 8px;
+      margin: 0 12px 8px 12px;
+      background: rgba(0, 0, 0, 0.75);
+      display: flex;
+      align-items: center;
+      --media-control-padding: 6px;
+    }
+
+    media-controller[breakpoint-sm] media-control-bar {
+      height: 46px;
+      --media-control-padding: 7px;
+    }
+
+    media-controller[breakpoint-md] media-control-bar {
+      height: 50px;
+      --media-control-padding: 8px;
+    }
+
+    media-captions-button:not(:is([media-captions-list], [media-subtitles-list])) {
+      display: none;
+    }
+
+    .live-controls-left {
+      margin-right: auto;
+    }
+
+    [disabled],
+    [aria-disabled='true'] {
+      opacity: 60%;
+      cursor: not-allowed;
+    }
```

Comment:
this CSS is also required for when features are not supported 
https://github.com/muxinc/media-chrome/blob/94ccd2ba6c44e6ee279f4dc5d8527456c5c0ac75/src/js/themes/micro.js#L18-L40

Reply:
Have added this and definitions for the optional extra buttons. Is there anything else that needs doing to accommodate the optional buttons in the theme templates? How do consumers of the theme enabled these extra buttons?

Reply:
oh sorry about this for the default disabled buttons the syntax changed a little. this should only be added, you can copy / paste that 

https://github.com/muxinc/media-chrome/blob/main/src/js/themes/microvideo.js#L224-L236